cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1151,22 +1136,6 @@ def qubit(self) -> raw_types.Qid:
     def _as_pauli_string(self) -> PauliString:
```

Comment:
I think this is no longer referenced now and can be removed.

Reply:
`__neg__` is actually using it, I can replace it to "_try_interpret_as_pauli_string" instead though.

Reply:
Oh, `__neg__` seems to do the same thing as the base implementation (but less efficiently) so may as well delete that here  too.

Reply:
Cool, amended the commit